Sources/PackageModel/PackageModel.swift

```diff
@@ -129,6 +129,12 @@ extension Package.Error: CustomStringConvertible {
     }
 }
 
+extension Manifest {
+    public var diambiguateByProductIDs: Bool {
```

Comment:
small typo here, should be `disambiguateByProductIDs`

Reply:
good catch :)